src/containers/ProfileDetails/components/ProviderId/components/FormModal/index.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Input, Form, Select } from 'antd';
+import Modal from 'components/Modal';
+
+const { Option } = Select;
+
+const FormModal = ({ form, visible, closeModal, onSubmit, fieldName, title }) => {
```

Comment:
just create the Form.useForm() inside the component, you don't need it as a prop